internal/entities/role.go

```diff
@@ -1,5 +1,38 @@
 package entities
 
+type UserRole string
+
+const (
+	UserRoleAdmin      UserRole = "Admin"
```

Comment:
I am not sure if those roles are helpful. Currently we need those roles to check if a person is from the tbz, digital agency or from the university.

The background is that we only want to link the person who is from the TBZ in the frontend to a watering plan.